datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validators/validator.py

```diff
@@ -251,9 +251,50 @@ def check_sysobjectids_are_duplicated(self, sysobjectids, profile):
                 self.duplicated[sysobjectid] = self.used_sysobjid[sysobjectid]
 
 
+class TableColumnHasTagValidator(ProfileValidator):
+    """
+    Validator responsible to check if all metrics related to a column in a profile
+    have tags to identify which row/entity they belong to
+    """
+
+    def validate(self, profile, path):
+        file_contents = find_profile_in_path(profile, path)
+        metrics = {}
+        if file_contents.get('metrics'):
+            metrics = file_contents.get('metrics')
+        for metric in metrics:
+            if metric.get('table'):
+                if not metric.get('metric_tags'):
+                    self.fail(
+                        "'metric_tags' is missing for table metric defined in line {}.".format(metric.get('__line__'))
+                    )
+                else:
+                    all_metric_tags_are_valid, lines = self.check_metric_tags_are_valid(metric.get('metric_tags'))
+                    if not all_metric_tags_are_valid:
+                        self.fail(
+                            "metric_tables defined in lines {} are not valid. \
+                        \nmetric_tags defined does not contains neither option 'column' nor 'index'".format(
```

Comment:
```suggestion
                            "metric_tables defined in lines {} are not valid. \
                        \nmetric_tags must have 'column' or 'index' value".format(
```
or is the rule that they must have both? If so, it should say "\nmetric_tags must have both 'column' and 'index' values"